lib/rust/ensogl/core/src/display/style/sheet.rs

```diff
@@ -191,11 +192,13 @@ impls! { From<&StaticPath> for Expression { |t| Self::reference(t) }}
 #[derive(Clone)]
 pub struct BoundExpression {
     args:     Vec<Index<Query>>,
+    #[allow(clippy::type_complexity)]
     function: Rc<dyn Fn(&[&Data]) -> Data>,
 }
 
 impl BoundExpression {
     /// Constructor.
+    #[allow(clippy::type_complexity)]
```

Comment:
is it really needed here? This type does not look so complex. If clippy warns here, maybe ithere is a setting for it in clippy conf now - is there one?

Reply:
There is a configuration parameter `type-complexity-threshold`, which we could raise.

Reply:
What is the status of this?

Reply:
Fixed on my branch. The default is `250`.